sklearn/mixture/base.py

```diff
@@ -194,29 +194,33 @@ def fit(self, X, y=None):
         max_log_likelihood = -np.infty
         self.converged_ = False
 
+        n_samples, _ = X.shape
         for init in range(n_init):
             self._print_verbose_msg_init_beg(init)
 
             if do_init:
                 self._initialize_parameters(X)
-            current_log_likelihood, resp = self._e_step(X)
+                self.lower_bound_ = np.infty
```

Comment:
@ngoix Sorry I forgot that - indeed. I don't know why I've missed your comment.
Sorry for that. I've solved that on #7180